testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/atom/AtomProviderTest.java

```diff
@@ -68,6 +71,14 @@
     public static Archive<?> deploy() {
         WebArchive war = TestUtil.prepareArchive(AtomProviderTest.class.getSimpleName());
         war.addClass(AtomProviderTest.class);
+
+        war.addAsManifestResource(PermissionUtil.createPermissionsXmlAsset(

```

Comment:
With https://github.com/resteasy/Resteasy/pull/1126 only permission
`new RuntimePermission("accessDeclaredMembers"))`
is required to make the test pass.